test/functional/plugin/lsp_spec.lua

```diff
@@ -5,29 +5,33 @@ local buf_lines = helpers.buf_lines
 local dedent = helpers.dedent
 local exec_lua = helpers.exec_lua
 local eq = helpers.eq
+local eq_dumplog = helpers.eq_dumplog
 local insert = helpers.insert
-local iswin = helpers.iswin
 local retry = helpers.retry
 local NIL = helpers.NIL
 
 -- Use these to get access to a coroutine so that I can run async tests and use
 -- yield.
 local run, stop = helpers.run, helpers.stop
 
-if helpers.pending_win32(pending) then return end
+-- Fake LSP server.
+local fake_lsp_code = 'test/functional/fixtures/fake-lsp-server.lua'
+local fake_lsp_logfile = 'Xtest-fake-lsp.log'
 
-local lsp_test_rpc_server_file = "test/functional/fixtures/lsp-test-rpc-server.lua"
-if iswin() then
-  lsp_test_rpc_server_file = lsp_test_rpc_server_file:gsub("/", "\\")
-end
+teardown(function()
+  os.remove(fake_lsp_logfile)
+end)
 
-local function test_rpc_server_setup(test_name, timeout_ms)
+local function fake_lsp_server_setup(test_name, timeout_ms)
   exec_lua([=[
     lsp = require('vim.lsp')
-    local test_name, fixture_filename, timeout = ...
+    local test_name, fixture_filename, logfile, timeout = ...
     TEST_RPC_CLIENT_ID = lsp.start_client {
+      cmd_env = {
+        NVIM_LOG_FILE = logfile;
```

Comment:
very weird: this works locally but fails on all CI jobs. If `cmd_env` is set to anything at all (even empty table), the fake LSP server fails to start. Difficult to see why in CI...

Reply:
````
INFO  2020-02-09T10:37:51.742 27914 emsg_multiline:687: E5113: Error while calling lua chunk: test/functional/fixtures/fake-lsp-server.lua:1: module 'vim.ls
p.protocol' not found:                                                                                                                                      
        no field package.preload['vim.lsp.protocol']                                                                                                        
        no file '~/.config/nvim/lua/vim/lsp/protocol.lua'                                                                                                   
...
```